conversation_reconstruction/dataflow/conversation_reconstruction/dataflow_clean_output.py

```diff
@@ -15,79 +15,77 @@
 
 -------------------------------------------------------------------------------
 
-A dataflow pipeline to collect statistics on Wikipedia talk pages including number of revisions and their sizes.
+A dataflow pipeline to clean scored outputs.
 
 Run with:
 
-  python dataflow_metadata.py --input [YourIngestedFiles] --output [IntendedOutputStorage] --setup_file ./setup.py
-         [Optional] --testmode (runnning locally for test)
+  python dataflow_clean_output.py --setup_file ./setup.py --input=InputStorage --output=OutputStorage --project=YourCloudProject --bucket=YourCloudBucket
```

Comment:
use '\' characters to make the command line max 80 char width and more readable.